sagemaker_neo_compilation_jobs/deploy_pytorch_model_on_Inf1_instance/pytorch_torchvision_neo_on_Inf1.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploying pre-trained PyTorch vision models with Amazon SageMaker Neo On Inf1 Instance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Amazon SageMaker Neo is API to compile machine learning models to optimize them for our choice of hardward targets. Currently, Neo supports pre-trained PyTorch models from [TorchVision](https://pytorch.org/docs/stable/torchvision/models.html). General support for other PyTorch models is forthcoming."
```

Comment:
Rephrase "Amazon SageMaker Neo is API to compile machine...:"

Take from our SM Dev Guide:
"Neo is a capability of Amazon SageMaker that enables machine learning models to train once and run anywhere in the cloud and at the edge."